### ISSUE TITLE
Show service name undex/over/next to serviceicon

### ISSUE THREAD

**Issue by Heidistein** [getferdi/ferdi#585]
I try to switch from RamBox to Ferdi. However I have several services with the same icon (slack, nextcloud). It would be a great help if I just cloud add a small textual label next to, under, over the icon. Don't care where.

Sorry that this is not a pull request, my JS/css knowledge is insufficient. I tried copying the tooltip to a div. Didn't get the correct tooltip and a misplaced div...

**Comment by kytwb** [comment]
@Heidistein Thank you for the feedback! 🙏

**Comment by Heidistein** [comment]
this is still relevant

**Comment by Heidistein** [comment]
It still is relevant :)

**Comment by Heidistein** [comment]
The issue is still relevant.

On January 30, 2021 4:59:54 AM GMT+01:00, "stale[bot]" <notifications@github.com> wrote:
>This issue has been automatically marked as stale because it has not
>had recent activity. Please check if this issue is still relevant and
>please close it if it's not. This will make sure that our open issues
>are actually of use and reduce the list of obsolete issues. Thank you
>for your contributions.
>
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/getferdi/ferdi/issues/585#issuecomment-770152264

-- 
Verstuurd vanaf mijn Android apparaat met K-9 Mail. Excuseer mijn beknoptheid.

**Comment by mfortini** [comment]
I agree, the main reason I use rambox/ferdi is that I want to keep services open with different accounts, so I need a way to distinguish them

**Comment by eminokic** [comment]
@Heidistein im unfamiliar with rambox but I believe I'm having the same issue. To clarify what is the problem you are having again. Is your goal to login to multiple accounts under the same service?

**Comment by vraravam** [comment]
This should actually be fairly simple (though I'm a developer, I'm not adept in css). If someone can raise a PR, I can test it and approve
